src/Aguacongas.Identity.Firestore/FirestoreTableNamesConfig.cs

```diff
@@ -0,0 +1,24 @@
+namespace Aguacongas.Identity.Firestore
+{
+    public class FirestoreTableNamesConfig
+    {
+        private struct Defaults
+        {
+            internal const string UsersTableName = "users";
```

Comment:
Can you use `static readonly` when possible instead of `const`, please

Reply:
What do you mean by
>Can you a test with custom collections ?

?

Reply:
add a test

Reply:
Added tests and changed to `static readonly`.